ci/compare-images.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+#
+# This script serves to compare two docker images using skopeo tool. This gives
+# a brief information regarding the following image differences:
+#   - size
+#   - architecture
+#   - operating system
+#   - config
+#     - default user
+#     - exposed ports
+#     - environment variables
+#     - entrypoint
+#     - working directory
+#     - labels
+#   - Python packages
+#   - RPM packages
+#
+# It uses the skopeo TODO downloads images locally...
+#
+# Local execution: ./ci/compare-images.sh <image-1> <image-2>
+#   Note: <image-*> is in the format <repository@sha256:SHA>
+#
+# Example usage:
+#    ./ci/compare-images.sh quay.io/opendatahub/workbench-images@sha256:e92bf20e127e545bdf56887903dc72ad227082b8bc23f45ff4f0fc67e6430318 ghcr.io/jiridanek/notebooks/workbench-images:base-ubi9-python-3.9-jd_ubi_base_adedd4a943977ecdcb67bc6eb9eda572d10c3ddc
+
+shopt -s globstar
```

Comment:
What about

```
set -Eeuxo pipefail
```

https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

to fail if something needed is missing?

I happen to not have skopeo, so

```
./ci/compare-images.sh quay.io/opendatahub/workbench-images@sha256:e92bf20e127e545bdf56887903dc72ad227082b8bc23f45ff4f0fc67e6430318 ghcr.io/jiridanek/notebooks/workbench-images:base-ubi9-python-3.9-jd_ubi_base_adedd4a943977ecdcb67bc6eb9eda572d10c3ddc
./ci/compare-images.sh: line 26: shopt: globstar: invalid shell option name
Gathering the metadata for the image: 'quay.io/opendatahub/workbench-images@sha256:e92bf20e127e545bdf56887903dc72ad227082b8bc23f45ff4f0fc67e6430318'
Image SHA: 'e92bf20e127e545bdf56887903dc72ad227082b8bc23f45ff4f0fc67e6430318'
./ci/compare-images.sh: line 42: skopeo: command not found
./ci/compare-images.sh: line 45: skopeo: command not found
```